github/src/main/kotlin/gropius/sync/github/model/TimelineEventInfo.kt

```diff
@@ -0,0 +1,21 @@
+package gropius.sync.github.model
+
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.index.Indexed
+import org.springframework.data.mongodb.core.mapping.Document
+import java.time.OffsetDateTime
+
+/**
+ * Mapping of a single timeline event from neo4j to github
+ */
+@Document
+data class TimelineEventInfo(
```

Comment:
missing documentation for properties

Reply:
cafe999d546dae9937ec2e811c2d873877c50a22